en/extension/work-with-vue.md

```diff
@@ -1,20 +1,18 @@
-# 在面板中使用 Vue
+# Working with Vue
 
-如果你已经掌握了 [编写面板界面](writing-your-panel.md) 这章中的界面编写方法，你或许会觉得这样
-编写界面有些繁琐。是否能够使用一些前端界面框架来提升界面编写效率呢？答案是肯定的。Cocos Creator
-支持任何界面框架如 [Vue](http://vuejs.org/)，[React](https://facebook.github.io/react/)，
-[Polymer](http://polymer-project.org/) 等等。
+If you have already mastered the interface writing method in the [writing panel interface](writing-your-panel.md) section, one might think that writing the interface is a bit cumbersome. Is it possible to use some front-end interface frameworks to improve the efficiency of interface writing? The answer is yes. Cocos Creator
+supports many interface frameworks such as [Vue](http://vuejs.org/), [React](https://facebook.github.io/react/),
```

Comment:
Is there a new line here? Why is a newline needed?

Reply:
not needed. Fixed. Thank you.